### PR TITLE
fix(desktop/wallet2) Updated wallet2 warning text

### LINKED ISSUE #3431: change icon for wallet v2 and update message/warnings
- update the icon for the wallet v2 to something else (anything) so it's less confusing
- update messages/descriptions so it's clear it's WIP and should not be used
-- in the advanced settings, the wallet v2 should say "DO NOT USE / under active development"

### PR DESCRIPTION
**Pull request by alexandraB99**
Also changed icon from wallet to cancel, to indicate
that the feature is incomplete

Closes #3431

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

File: ui/app/AppMain.qml [hunk 1]
```diff
@@ -364,7 +364,8 @@ Item {
                 },
 
                 StatusNavBarTabButton {
-                    icon.name: "wallet"
+                    //TODO temporary icon name, switch back to wallet
+                    icon.name: "cancel"
```
Comment (alaibe) [inline ui/app/AppMain.qml:368]: This is clear that way: DO NOT USE :)

File: ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml [hunk 2]
```diff
@@ -289,7 +289,8 @@ ScrollView {
             height: 310
             showCancelButton: true
             //% "This feature is experimental and is meant for testing purposes by core contributors and the community. It's not meant for real use and makes no claims of security or integrity of funds or data. Use at your own risk."
-            confirmationText: qsTrId("this-feature-is-experimental-and-is-meant-for-testing-purposes-by-core-contributors-and-the-community--it-s-not-meant-for-real-use-and-makes-no-claims-of-security-or-integrity-of-funds-or-data--use-at-your-own-risk-")
+            confirmationText: (settingsProp === "isWalletV2Enabled" ? qsTr("<b>--DO NOT USE - UNDER ACTIVE DEVELOPMENT--</b>\n") : "") +
+                qsTrId("this-feature-is-experimental-and-is-meant-for-testing-purposes-by-core-contributors-and-the-community--it-s-not-meant-for-real-use-and-makes-no-claims-of-security-or-integrity-of-funds-or-data--use-at-your-own-risk-")
```
Comment (anastasiyaig) [inline ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml:293]: there is no point to say `DO NOT USE` and `use at your own risk` in the same statement

**Review by anastasiyaig — CHANGES_REQUESTED** [review]

**Comment by anastasiyaig** [comment]
Screenshot 2021-09-09 at 17 41 23

we can leave only header which is in bold now and remove the text after it since it contradicts with the header (do not use and use at own risk)

**Comment by alexandraB99** [comment]
> Screenshot 2021-09-09 at 17 41 23
> 
> we can leave only header which is in bold now and remove the text after it since it contradicts with the header (do not use and use at own risk)

@anastasiyaig so you mean to remove all text from "This feature..." to "Use at your own risk" and leave only the title "CONFIRM YOUR ACTION" and "--DO NOT USE - UNDER ACTIVE DEVELOPMENT--" ?  @iurimatias what's your opinion on this?

**Comment by iurimatias** [comment]
I think ti's fine as it is, I'll make a commit with a slight change and then merge

**Pull request merged**